### PR TITLE
Adds feedback mechanism and more site visual improvements

### PR DESCRIPTION
**Pull request by kolchfa-aws**
Adds feedback mechanism, adds max width to the layout, improves the visual look of copy buttons and back to top button.

### Checklist
- [ x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by hdhalter — APPROVED** [review]

**Review by ariamarble — APPROVED** [review]

File: _layouts/default.html [hunk 1]
```diff
@@ -208,9 +202,13 @@ <h2 class="text-delta">Table of contents</h2>
...
       <div class="toc-wrap">
-        <div class="toc">
-          {% include toc.html html=content h_min=2 h_max=2 class="toc-list" item_class="toc-item" sanitize=true %}
+        <div class="sticky-wrap">
+          <div class="toc">
+            {% include toc.html html=content h_min=2 h_max=2 class="toc-list" item_class="toc-item" sanitize=true %}
+          </div>
+          {% include feedback.html %}
         </div>
+        {% comment %}{% include feedback.html %}{% endcomment %}
```
Comment (AMoo-Miki) [inline _layouts/default.html:211]: Why do we have commented out content?

File: assets/js/feedback.js [hunk 2]
```diff
@@ -2,12 +2,31 @@ const feedbackButtons = document.querySelectorAll('.feedback-button');
...
+        // find the hidden feedback text
+        sibling = button.nextElementSibling;
+
+        while (!(sibling.classList.contains('text-small') && sibling.classList.contains('hidden'))) {
+            sibling = sibling.nextElementSibling;
+        }
+        // show the hidden feedback text
+        if (sibling != null) {
+            sibling.classList.remove('hidden');
+        }
```
Comment (AMoo-Miki) [inline assets/js/feedback.js:17]:
```suggestion
        const parent = button.parentElement;
        const index = [...parent.children].indexOf(button) + 1;
        const sibling = parent.querySelector(`:nth-child(${index}) ~ .text-small.hidden`);
        sibling?.classList.remove('hidden);
```

File: assets/js/feedback.js [hunk 3]
```diff
@@ -2,12 +2,31 @@ const feedbackButtons = document.querySelectorAll('.feedback-button');
...
+        var buttonSibling;
+        if (button.id === 'yes') {
+            buttonSibling = button.nextElementSibling;
+        }
+        else {
+            buttonSibling = button.previousElementSibling;
+        }
+        if (buttonSibling != null) {
+            buttonSibling.disabled = true;
+        }
```
Comment (AMoo-Miki) [inline assets/js/feedback.js:30]: Shouldn't this be `buttonSibling.disabled = false`? Also

```suggestion
        button[button.id === 'yes' ? 'nextElementSibling' : 'previousElementSibling']?.disabled = false;

File: assets/js/link-feedback.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+const feedbackIssueLinks = document.querySelectorAll('.feedback-issue');
```
Comment (AMoo-Miki) [inline assets/js/link-feedback.js:1]: Looks like we are adding listeners to everything. I would propose dropping all of the listeners that are being added throughout the code and instead add one giant click-listener to the document and then check the `target` within the listener's callback to determine what to do. As in
```
document.addEventListener('click', e => {
    const { target } = e;
    if (target.matches('.feedback-issue')) return gtag('event', 'submit_issue_click');
    if (target.matches('.feedback-edit')) return return gtag('event', 'edit_page_click');
    if (target.matches('.copy-label')) return copyLabel(target);
    if (target.matches('.copy-curl-label')) return copyLabel(target, true);
});

File: _layouts/default.html (hunk 1 above)
Comment (kolchfa-aws) [reply _layouts/default.html:211]: Removed.

File: assets/js/feedback.js (hunk 3 above)
Comment (kolchfa-aws) [reply assets/js/feedback.js:30]: It should disable both buttons, so should be disabled=true

File: assets/js/link-feedback.js (hunk 4 above)
Comment (kolchfa-aws) [reply assets/js/link-feedback.js:1]: That's a good idea!

**Pull request merged**